src/combinatorics/simple_graph/strongly_regular.lean

```diff
@@ -123,38 +124,51 @@ begin
   { apply hnw, rwa adj_comm, },
 end
 
-lemma strongly_regular_complement (n k l m : ℕ) (h : G.is_strongly_regular_of n k l m) :
+lemma compl_regular_of_strongly_regular (h : G.is_strongly_regular_of n k l m) :
+  Gᶜ.is_regular_of_degree (n - k - 1) :=
+begin
+  rw [← h.card, nat.sub_sub, add_comm, ←nat.sub_sub],
+  exact G.is_regular_compl_of_is_regular k h.regular,
+end
+
+lemma compl_adj_common_of_strongly_regular (h : G.is_strongly_regular_of n k l m) :
+  ∀ (v w : V), Gᶜ.adj v w → fintype.card ↥(Gᶜ.common_neighbors v w) = n - (2 * k - m) - 2 :=
```

Comment:
Make sure to move arguments to before the colon and remove the `intros` first line.